tests/all_funcs

```diff
@@ -6,9 +6,9 @@ function proceed {
 }
 
 make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_BER'; proceed
-make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_SAX'; proceed
 make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_FNV'; proceed
-make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_OAT'; proceed
 make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_JEN'; proceed
-make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_MUR'; proceed
-make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_SFH'; 
+make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_OAT'; proceed
+make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_SAX'; proceed
+make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_SFH'; proceed
+make clean tests_only EXTRA_CFLAGS='-DHASH_FUNCTION=HASH_MUR -DHASH_USING_NO_STRICT_ALIASING -fno-strict-aliasing'
```

Comment:
Where do you see `MUR_CFLAGS` being applied?
Also, what OS/toolchain are you using? Maybe there's some difference between the behavior of Make on OSX and (whatever) on (whatever). Because both before and after this change, I'm seeing that the *only* Makefile targets that involve `MUR_CFLAGS` are `MUR` itself (known after this patch as `keystat`) and `hash_scan`. The only *other* way to trigger a production involving `MUR_CFLAGS` is to set the Makefile variable `HASH_FUNCTION` to `HASH_MUR`, and I don't believe either of us has ever done that.